src/App.vue

```diff
@@ -43,7 +43,7 @@ watch(modalOpen, val => {
   >
     <UiLoading v-if="app.loading || !app.init" class="overlay big" />
     <div v-else>
-      <MainSidebar />
+      <TheSidebar />
```

Comment:
LeftSidebar maybe?

Reply:
That's easy to confuse with the layout sidebar. It's "the" sidebar, because there will only be one.